frontend/apps/reader/readerui.lua

```diff
@@ -106,6 +106,11 @@ function ReaderUI:init()
     -- cap screen refresh on pan to 2 refreshes per second
     local pan_rate = Screen.low_pan_rate and 2.0 or 30.0
 
+    if Device:isAndroid() then
```

Comment:
I don't think this makes sense?

Reply:
What does not make sense? The `isAndroid` check?

Reply:
In practice of course, `Device:setIgnoreInput(true)` is a no-op on systems other than Android. But if another platform were to make it do something in the future, we'd want it to Just Work™ after providing a platform-specific implementation without having to delete all the isAndroid checks. ;-)

Reply:
Oh it does make sense. On opening a large book (Galunids python.epub) I can provoke ANRs on my mobile. With this in, the ANR is out.

Reply:
> What does not make sense? The isAndroid check?

Exactly. It seems completely antithetical to the Device abstraction.

Reply:
See https://github.com/koreader/koreader/pull/8501#discussion_r762220904 above. (Remove if isAndroid, and add a comment.)

Reply:
And if `if Device:isAndroid()` were indeed necessary, we'd want it to say something like `if Device:requiresIgnoreInput()` for the same reason as stated above.

Occasionally there are some things that are truly Android specific workarounds, but conceptually speaking this doesn't look like one of them.